src/options/printer_options.toml

```diff
@@ -40,14 +40,6 @@ name   = "Printing"
   default    = "false"
   help       = "when printing unsat cores, include unlabeled assertions"
 
-[[option]]
-  name       = "flattenHOChains"
-  category   = "expert"
-  long       = "flatten-ho-chains"
-  type       = "bool"
-  default    = "false"
-  help       = "print (binary) application chains in a flattened way, e.g. (a b c) rather than ((a b) c)"
```

Comment:
our parser no long has the limitation of not being able to parse ((a b) c)? That was the motivation of having this option in the first place, so that we could generate benchmarks that cvc5 could parse.